contribs/gmf/src/raster/widgetComponent.html

```diff
@@ -16,12 +16,12 @@
       <span ng-show="ctrl.elevationLoading" class="fa custom-spinner-generic">
           <%=require('gmf/icons/spinner.svg?viewbox&height=1rem')%>
       </span>
-      <span ng-show="!ctrl.elevationValue && !ctrl.elevationLoading" translate>Elevation…</span>
+      <span ng-show="!ctrl.elevationValue && !ctrl.elevationLoading" translate>Raster</span>
```

Comment:
=> ?
```
      <span ng-show="ctrl.elevationLoading || !ctrl.elevationValue" class="fa custom-spinner-generic">
          <%=require('gmf/icons/spinner.svg?viewbox&height=1rem')%>
      </span>
```

Reply:
I'll need words...

Reply:
?

Reply:
I don't understand what you means with `=> ?`

Reply:
If you mean "I've said that it's better to always have a loading spinner", I've replied in the first comment of the PR

Reply:
OK I understand better =>
+ For Raster when the mouse is out of the map
But actually, it's strange to have when we move the mouse on the maps
'Elevation...' then loading icon then the value
For me, we should just have the loading icon then the value

Reply:
Yes but it's strange to have the loading symbol forever when you are on tools panel or on the layertree... It looks like a bug.
I don't know... we need a third opinion @ybolognini @arnaud-morvan @RenataMuellerC2C ?

Reply:
> Yes but it's strange to have the loading symbol forever when you are on tools panel or on the layertree... It looks like a bug.

On my last comment I don't say that... (+1 For Raster when the mouse is out of the map)

Reply:
Ok, I'll display the 'loading spinner' always on the map when it has no value. And `Raster` when the mouse pointer is not on the map.